std/range/package.d

```diff
@@ -3205,18 +3200,42 @@ private:
     else
         alias fun = Fun[0];
 
+    import std.traits: ReturnType, functionAttributes, FunctionAttribute;
+    enum returnByRef_ = (functionAttributes!fun & FunctionAttribute.ref_) ? true : false;
+    static if(returnByRef_)
+        ReturnType!fun *elem_;
```

Comment:
Space between required `if (`. This is why Travis complains.

Likewise, at least one if and one foreach below.